src/routers/product-router.js

```diff
@@ -1,4 +1,8 @@
 const express = require('express');
 const router = express.Router();
 
+const { ProductModel } = require('../models');
```

Comment:
no need to pass model to function, here we should attach only req handler function